ROR-Assignment-Day-01/check_school_level.rb

```diff
@@ -0,0 +1,22 @@
+def check_school_level(grade)
+  if 1<= grade && grade <= 5
+    return "Elementary"
+  elsif 6<= grade && grade <= 8
+    return "Middle School"
+  elsif 9<= grade && grade <= 12
+    return "High School"
+  else
+    return "College"
+
+  end
+end
+
+print "Enter a School Grade:"
+
+grade = gets.strip
+
+if grade.match(/^\d+$/)
```

Comment:
As this is conditional statement so i think it's better to check like this
```suggestion
if grade.match?(/^\d+$/)
```